### PR TITLE
feat: data access role

### PR DESCRIPTION
**Pull request by jjfrench**
Optional: Creates a data-access-role if the user doesn't already have one - can be used for the veda-stac-ingestor and veda-data-pipelines deployments
- adds in the ability for delta-backend-staging-*-external-role (once deployed) to assume this data-access-role by defining DELTA_BACKEND_EXTERNAL_ROLE_ARN in .env
- not defining BUCKETS in .env will not create this role

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,12 @@ User groups with pre defined roles can be creating by providing the existing rol
 2. CDK deploy change and note `veda-auth-stack-<STAGE>.userpoolid` in output. It will include the deployment region and a UUID, for example `us-west-2:11111111-1111-1111-1111-111111111111`
 3. Add a new statement to the role's trust policy in the AWS IAM console. Navigate to the desired role, choose `Trust Relationship` and select `edit`--be careful to preserve the existing trust statements when appending a new statement for this identity pool.
 
+## Creating a data access role
```
Comment (emileten) [inline README.md:48]: Thanks for adding documentation. In the case of adding the data access role, I was wondering how you operated things, i.e, what you actually _did_ on your terminal to create that role. Did you just follow the steps you outlined below and then ran `cdk deploy` ?

**Review by emileten — APPROVED** [review]
Another more general question about this repo is that, after reading the beginning of the README I had the impression that this repo is only about cognito related stuff 

> This codebase represents the Cognito-based authentication system used for the VEDA project.

Whereas we also create other things, such as this data access role. I feel we could change this first README line to reflect this fact, or it's me being new to AWS and saying something incorrect!

File: README.md (hunk 1 above)
Comment (jjfrench) [reply README.md:48]: Right, the same `cdk deploy --profile [profile-name]` command after setting up the environment configuration file. Sorry if I misinterpreted your previous question about documentation

Comment (emileten) [reply README.md:48]: Nope all good both answers were helpful

**Comment by emileten** [comment]
Maybe we can merge this branch ? It looks like we need either of Aimee's or Alexandra's review because I don't have write access.

**Comment by anayeaye** [comment]
@jjfrench @abarciauskas-bgse I'm wondering if this might work as a separate tool in the same project. It seems like it shouldn't be necessary to deploy a cognito auth stack in order to create and/or manage bucket policies for a data access role. Perhaps we could look at adding a separate stack for role management? In veda-backend I made a [separate directory for the VPC deployment ](https://github.com/NASA-IMPACT/veda-backend/tree/develop/standalone_base_infrastructure) which we manage independently of the services. The sub directory isn't obvious and might not be the best pattern to follow but it's something to discuss.

**Pull request closed without merging**